rllib/agents/impala/vtrace_torch_policy.py

```diff
@@ -209,7 +208,7 @@ def make_time_major(policy, seq_lens, tensor, drop_last=False):
         T = tensor.shape[0] // B
     else:
         # Important: chop the tensor into batches at known episode cut
-        # boundaries. TODO(ekl) this is kind of a hack
+        # boundaries.
```

Comment:
Isn't it still a hack? Please restore the comment.

Reply:
I don't think it's a "hack", since IMPALA explicitly indicates through the divisibility requirement that the batch must be divisible by rollout_fragment_length.